### PR TITLE
Update for Agent v3.50.0

### PR DESCRIPTION
**Pull request by DrJosh9000**

### CODE REVIEW

File: data/content/agent_attributes.yaml [hunk 1]
```diff
@@ -127,6 +127,12 @@ attributes:
   required: false
   desc: |
       The directory to use for agent hooks. See [Hook locations](https://buildkite.com/docs/agent/v3/hooks#hook-locations) for details.
+- name: job-log-path
+  env_var: BUILDKITE_JOB_LOG_PATH
+  default_value: "platform's TempDir; e.g. '/tmp'"
+  required: false
+  desc: |
+      The directory of the temporary job log file created by the `enable-job-log-tmpfile` [agent configuration option](/docs/agent/v3/configuration). Introduced in [v3.50.0](https://github.com/buildkite/agent/releases/tag/v3.50.0).
```
Comment (mbelton-buildkite) [inline data/content/agent_attributes.yaml:135]: It looks like the "configuration option" link is missing the H2 heading. I think it should be:

```suggestion
      The directory of the temporary job log file created by the `enable-job-log-tmpfile` [agent configuration option](/docs/agent/v3/configuration#enable-job-log-tmpfile). Introduced in [v3.50.0](https://github.com/buildkite/agent/releases/tag/v3.50.0).
```

File: pages/agent/v3/help/_lock_release.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ script.
 
 ### Usage
 
-`buildkite-agent lock release [key]`
+`buildkite-agent lock release [key] [token]`
 
 ### Description
```
Comment (mbelton-buildkite) [inline pages/agent/v3/help/_lock_release.md:19]: A suggestion you don't have to address here, but I think the description should mention how `token` fits in.

**Review by mbelton-buildkite — COMMENTED** [review]
One link that I think needs updating, then looks good :)

**Review by moskyb — APPROVED** [review]

File: pages/agent/v3/help/_lock_release.md (hunk 2 above)
Comment (DrJosh9000) [reply pages/agent/v3/help/_lock_release.md:19]: Done.

**Pull request merged**